acceptance/acceptance_test.go

```diff
@@ -225,6 +225,15 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		t.Skipf("Disabled via Cloud setting in %s (CLOUD_ENV=%s)", configPath, cloudEnv)
 	}
 
+	if config.RequiresStageFeature(UnityCatalogEnabledWorkspace) {
+		if os.Getenv("TEST_METASTORE_ID") == "" {
```

Comment:
Do we need to set this variable even if we run the acceptance test locally / on CI as part of unit tests?

Reply:
I've changed the check to be only be done when Cloud=true